client/app/src/main/java/com/example/protego/DoctorViewPatientSelections.java

```diff
@@ -20,7 +24,13 @@ protected void onCreate(Bundle savedInstanceState) {
         connectButtonToActivity(R.id.medicationButton, DoctorViewPatientMedication.class);
         connectButtonToActivity(R.id.notesButton, DoctorViewPatientNotes.class);
         connectButtonToActivity(R.id.vitalsButton, DoctorViewPatientVitals.class);
+        Bundle extras = getIntent().getExtras();
+        patientName = extras.getString("patientFirst");
```

Comment:
Patient last names are asked during registration so we should be able to display their full name now.